ui/tests/acceptance/enterprise-namespaces-test.js

```diff
@@ -88,4 +89,90 @@ module('Acceptance | Enterprise | namespaces', function (hooks) {
       'Does not prepend root to namespace'
     );
   });
+
+  module('auth form', function (hooks) {
```

Comment:
I don't think I've seen modules nested in other modules before. Neat idea!

Reply:
I copied @hashishaw 's idea from the pki tests, thought it was a neat way to organize. (And make the setup reusable if/when we add more namespace tests?) : https://github.com/hashicorp/vault/blob/main/ui/tests/acceptance/pki/pki-engine-workflow-test.js#L154